docs/detections/alerts-view-details.asciidoc

```diff
@@ -123,33 +181,114 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
 - `url.full`: The full URL of the event source
 - `registry.path`: The full registry path, including the hive, key, and value
 
-If these fields aren't available, {elastic-sec} does not perform the query and does not display threat intelligence data for the alert.
+The Threat intelligence overview shows two types of matched indicators:
+
+* **Threat match detected**: Shows matched indicators that were discovered in the alert document of an alert generated from an indicator match rule. If none exist, the total number of matched indicators is zero.
+
+* **Fields enriched with threat intelligence**: Shows matched indicators that were discovered in the alert document of an alert that _wasn't_ generated from an indicator match rule. If none exist, the total number of matched indicators is zero.
+
+[discrete]
+[[expanded-threat-intel-view]]
+==== Expanded threat intelligence view
+
+From the right panel, click **Threat intelligence** to open the expanded Threat intelligence view within the left panel. 
+
+NOTE: The expanded threat intelligence view queries indices specified in the `securitySolution:defaultThreatIndex` advanced setting. Refer to <<update-threat-intel-indices, Update default Elastic Security threat intelligence indices>> to learn more about threat intelligence indices. 
+
+[role="screenshot"]
+image::images/expanded-threat-intelligence-view.png[Expanded view of threat intelligence on the alert, 70%]
+
+The expanded Threat intelligence view shows individual indicators within the alert document. You can expand and collapse indicator details by clicking the arrow button at the end of the indicator label. Each indicator is labelled with values from the `matched.field` and `matched.atomic` fields and displays the threat intelligence provider. 
+
+Matched threats are organized into two sections, described below. Within each section, matched threats are shown in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.
+
+**Threat match detected**
+
+The Threat match detected section is only populated with data if you're examining an alert that was generated from an indicator match rule. Indicator matches are logged in this section when certain field values in the alert document match with threat intelligence data you've ingested. 
+
+**Fields enriched with threat intelligence**
+
+Threat intelligence can also be found on alerts that weren't generated from indicator match rules. To make this data discoverable, {elastic-sec} enriches fields containing threat intelligence. Enriched fields are appended with `threat.indicator`. For example, if an alert document contains the `file.hash.md5` field, and the field's value matches a known indicator, {elastic-sec} appends the `file.hash.md5` field with `threat.indicator`.
```

Comment:
Also not really understand what this section means.

From reading this, I understood that we enrich alert documents with threat.indicator field, but that's not what happened.

This enrichment is real-time and doesn't change alert document

We use threat.indicator to make a match with alert fields

<img width="933" alt="Screenshot 2023-09-08 at 13 04 54" src="https://github.com/elastic/security-docs/assets/7609147/26ca419d-5227-4b9b-b5f4-d5602e5644a5">

Reply:
Here some [dev docs ](https://docs.elastic.dev/security-solution/dev-docs/detections/threat-intel/backend#investigation-time-enrichments) which we have 

The process is described in this document:
>gather threat intelligence on an alert, {elastic-sec} queries alert documents from the past 30 days and searches for fields that contain known threat intelligence. {elastic-sec} looks for the following fields:..

About this quote
>Elastic Security appends the file. hash.md5 field with threat. indicator

Yes, we query data from the threat indicators index. Then we don't change alerts actually, we just show on flyout information about this threat indicator. 
So we get info from this field - `threat.indicator` , but it not present in alert

Reply:
Thanks for linking to the additional docs, @nikitaindik. If I'm understanding the dev docs correctly, alert docs are queried for the same indicator fields, regardless of the alert type. The main differences seems to be when the query is executed and how/where the query results persist. I'm not sure how important it is to share this detail, but it might still be useful to describe the query at a high-level. How's this:

```suggestion
Threat intelligence can also be found on alerts that weren't generated from indicator match rules. {elastic-sec} queries the alert document for fields that contain known indicators and displays matched indicators in this section. 
```

Reply:
yes, the proposal changes look better from my perspective!